packages/framework-types/src/envelope.ts

```diff
@@ -64,6 +67,11 @@ export interface GraphQLOperation {
   variables?: Record<string, any>
 }
 
+export interface ConnectionDataEnvelope {
+  expirationTime: number // In Epoch format
```

Comment:
Epoch is not a format, but a day

```suggestion
  expirationTime: number // Since Unix epoch
```

Reply:
In the DynamoDB documentation, they refer to it as a format, [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-before-you-start.html#time-to-live-ttl-before-you-start-formatting)

Also, I think that when someone says `epoch format`, it is understood as "the number of seconds since epoch". What do you think?

Reply:
I see, Wikipedia says that people sometimes refer to it as "Epoch Time" which can be seen as a format. Leave it like this! :) 

Source https://en.wikipedia.org/wiki/Unix_time